document_page_tag/models/document_page_tag.py

```diff
@@ -15,10 +15,11 @@ class DocumentPageTag(models.Model):
         ("unique_name", "unique(name)", "Tags must be unique"),
     ]
 
-    @api.model
-    def create(self, vals):
+    @api.model_create_multi
+    def create(self, vals_list):
         """Be nice when trying to create duplicates"""
-        existing = self.search([("name", "=ilike", vals["name"])], limit=1)
-        if existing:
-            return existing
-        return super().create(vals)
+        for vals in vals_list:
+            existing = self.search([("name", "=ilike", vals["name"])], limit=1)
+            if existing:
+                return existing
+        return super().create(vals_list)
```

Comment:
**Issue**: When trying to create two tags at the same time, one that exists and one that doesn't, we return the one tag that does exist and don't create the one that doesn't.

Reply:
Changes included to handle tag creation with one exist and not exist at the same time